lib/rest-api/lambdas/delete_graph_request.py

```diff
@@ -11,35 +11,41 @@
 graph = Graph()
 user = User()
 
+def format_graph_name(graph_name):
```

Comment:
Just a suggestion and not necessarily a deal breaker but to reduce duplication, perhaps we could move this to a python util file and reference it from both the add_graph_request and delete_graph_request lamdas?